apps/api/src/shared/database/migrations/20231102181211-create-user-table.ts

```diff
@@ -11,6 +11,7 @@ export class CreateUserTable extends Migration {
       .createTable(TABLE_NAME, (table) => {
         table.increments('id').primary();
         table.string('uuid').notNullable().unique();
+        table.string('identity_provider_id').unique();
```

Comment:
We can mark this as nullable

Reply:
I think it is nullable by default. Did you mean notNullable?